### PR TITLE
Morse: add triggering for 'morse' & 'morse code'

### LINKED ISSUE #3193: Morse: Bad example queries
The translation from morse does not trigger when I type in "morse ... --- ..."

IA Page:  http://duck.co/ia/view/morse 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @und3f

### PR DESCRIPTION
**Pull request by kirkins**
## Related Issues and Discussions

Fixes #3193

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/Morse

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
This looks fine to me. I just want it deployed to beta before merge. Assigning to @moollaza

File: t/Morse.t [hunk 1]
```diff
@@ -37,12 +37,12 @@ ddg_goodie_test(
...
+    'morse hello, duck'             => build_test($duck, $duck, "hello, duck"),
+    'morse SOS'                     => build_test($sos, $sos, "SOS"),
+    'morse code SOS'                => build_test($sos, $sos, "SOS"),
+    'morse code hello, duck'        => build_test($duck, $duck, "hello, duck"),
     'SOS morse'                     => undef,
     'SOS morse code'                => undef,
-    'morse hello, duck'             => undef,
-    'morse code hello, duck'        => undef,
     'morse code for cheatsheet'     => undef,
     'morse code for cheat sheet'    => undef,
```
Comment (kirkins) [inline t/Morse.t]: Should we make any of the above 4 trigger too?

Comment (pjhampton) [inline t/Morse.t]: Yeah, I suppose we could open up the triggering 👍

Comment (kirkins) [inline t/Morse.t]: Ah I just misunderstood normally 'morse code for *' works it's just that 'cheat sheet' is an exception. 

I'm thinking we won't want to trigger things that end with "Morse" as it's a last name.

But I'll add an end trigger for queries like "SOS morse code".

Comment (kirkins) [inline t/Morse.t]: But then we'd trigger on something like "What is morse code" hmmm 🤔

Comment (kirkins) [inline t/Morse.t]: I think I will leave this one as is for now 😊. 

I have some other triggers I want to work on that I think will be a better use of time.

**Comment by moollaza** [comment]
@kirkins @pjhampton I think this may have been the behaviour before and it was removed. We've moved away from generic start triggers in many IAs because they cause problems.

I just did a quick check of the queries and every single query that starts with "morse" or "morse code" is _not_ an attempt to translate something into morse code. 

So I think on those grounds we shouldn't take this PR an further as it will create a lot of false-positive results.

**Comment by kirkins** [comment]
@moollaza That was [what I said originally](https://github.com/duckduckgo/zeroclickinfo-goodies/issues/3193#issuecomment-307651571).

First time I work on something told to do something with the high priority label. So start working on converting goodies to structured answer. As soon as I put in significant work [you remove the label](https://github.com/duckduckgo/zeroclickinfo-goodies/issues/1163#event-1022900272). Then pull requests never get resolved.

Seems target the target is always moving with you.

**Comment by moollaza** [comment]
@kirkins my apologies. It seemed like the intent with the "morse" trigger was for converting words from English to Morse code. I see now that @pjhampton's intent was to allow it for converting only Morse code to English, which should be OK. Right now it looks like the PR requries more work to ensure that though, and in its current state would result in a lot of over-triggering and irellevant results.

>First time I work on something told to do something with the high priority label. So start working on converting goodies to structured answers. As soon as I put in significant work you remove the label. 

I'm sorry about this as well. DuckDuckHack has been undergoing many changes lately, specifically with regards to what our highest priorities and the current focus is. As you're aware there are many open issues, several which are very old and have the "Priority: High" label. 

Many of these open issues are still valid and would be nice to have fixed, but the truth is that fixing most of them will have almost no impact. For example, the morse code Instant Answer is very unpopular and triggers on average < 5 times/day. Both of our time and effort is better spent on more important issues, like the ones found on the Issues Page, which are affecting hundreds and thousands of users per day (for example, queries related to currency conversions).

We recently tried to change what developers were focusing on, by creating the [Issues page](https://duckduckhack.com/issues) on DuckDuckHack.com and funnelling new devs towards that, and the issues with the "Highest Impact Tasks" label. Those issues are the issues we really want developers working on. @pjhampton and I rewrote much of the onboarding docs to try and ensure developers would see those, so they wouldn't waste time working on other less important issues.

It's our fault for not auditing all the stale issues. We should have removed the "High Priority" label from most of those. Sorry.

As a volunteer, I fully understand your frustration in giving us your time and having it go to waste. On the flip side, I'm responsible for more than just the DDH pull requests and I can't review everything that's incoming in a timely fashion, so I have to focus on the PRs that are absolutely necessary and high-impact.

**Pull request closed without merging**